_posts/2024-08-14-running-a-hsm-realm.md

```diff
@@ -0,0 +1,221 @@
+---
+layout: post
+title: "Running a Juicebox HSM realm"
```

Comment:
Maybe "Running a Juicebox hardware realm"? We can/do elaborate on the HSMs bit later, but the previously blog post names them that way